apps/ima/translations/src/en/pages.json

```diff
@@ -63,29 +60,31 @@
     "header": "Declaration"
   },
   "immigration-adviser-declaration": {
+    "adviser": "Immigration adviser declaration",
```

Comment:
I don't think this is needed here, the title is already showing as 'Declaration'

Reply:
Removed

Reply:
This has not been removed

Reply:
Now removed